compliance/compliance-server/src/main.rs

```diff
@@ -102,5 +107,11 @@ async fn check_wallet_compliance(
     }
 
     // 2. If not present, check the chainalysis API
-    Ok(ComplianceStatus::Compliant)
+    info!("address not cached in DB, querying Chainalysis");
+    let compliance_status = query_chainalysis(&wallet_address, chainalysis_api_key).await?;
+    let compliance_entry = compliance_status.as_compliance_entry();
```

Comment:
nit: i'm sure this is in the next PR, but `query_chainalysis` will return a `RiskAssessmentResponse`, not a `ComplianceStatus`. I think it makes sense for it to return a `ComplianceEntry`, as this code path accomplishes, by calling `ComplianceEntry::new(resp)` in `query_chainalysis`

Reply:
sure that works